lib/porro/model.rb

```diff
@@ -39,7 +39,7 @@ def self.included(base)
     end
 
     def initialize(params = {})
-      self.attributes = params
+      self.attributes = attributes.merge(params.to_h)
```

Comment:
agree that it is a user error - we could add a workaround maybe in the shared specs?